jdaviz/configs/default/plugins/data_menu/data_menu.vue

```diff
@@ -1,43 +1,74 @@
 <template>
   <div>
-    <div v-if="Object.keys(viewer_icons).length > 1" class="viewer-label invert-if-dark">
-      <j-tooltip span_style="white-space: nowrap">
-        <j-layer-viewer-icon span_style="float: right;" :icon="viewer_icons[viewer_id]"></j-layer-viewer-icon>
-      </j-tooltip>
-      <span class="invert-if-dark" style="margin-left: 24px; margin-right: 32px; line-height: 24px">{{viewer_reference || viewer_id}}</span>
+    <div v-if="Object.keys(viewer_icons).length > 1" class="viewer-label">
+      <span style="float: right;">
+        <j-layer-viewer-icon-stylized
+          tooltip=""
+          :label="viewer_id"
+          :icon="viewer_icons[viewer_id]"
+          :visible="true"
+          :is_subset="false"
+          :colors="['#939393']"
+          :linewidth="0"
+          :cmap_samples="cmap_samples"
+          btn_style="margin-bottom: 0px"
+        />
+      </span>
+      <span class="invert-if-dark" style="margin-left: 30px; margin-right: 36px; line-height: 28px">{{viewer_reference || viewer_id}}</span>
     </div>
 
-    <div v-for="(layer_info, layer_name) in visible_layers" class="viewer-label invert-if-dark">
-      <j-tooltip span_style="white-space: nowrap">
-        <j-layer-viewer-icon span_style="float: right;" :icon="layer_icons[layer_name]" :linewidth="layer_info.linewidth" :linestyle="'solid'" :color="layer_info.color"></j-layer-viewer-icon>
-      </j-tooltip>
-      <span class="invert-if-dark" style="margin-left: 24px; margin-right: 32px; line-height: 24px">
-        <v-icon v-if="layer_info.prefix_icon" dense>
-          {{layer_info.prefix_icon}}
-        </v-icon>
-        {{layer_name}}
-      </span>
+    <div v-for="item in layer_items.slice().reverse()" class="viewer-label">
+      <div v-if="item.visible">
+        <span style="float: right;">
+          <j-layer-viewer-icon-stylized
+            tooltip=""
+            :label="item.label"
+            :icon="item.icon"
+            :visible="item.visible"
+            :is_subset="item.is_subset"
+            :colors="item.colors"
+            :linewidth="item.linewidth"
+            :cmap_samples="cmap_samples"
+            btn_style="margin-bottom: 0px"
+          />
+        </span>
+        <span class="invert-if-dark" style="margin-left: 30px; margin-right: 36px; line-height: 28px">
+          <v-icon v-if="item.subset_type == 'spatial'" dense>
+            mdi-chart-scatter-plot
+          </v-icon>
+          <v-icon v-else-if="item.subset_type == 'spectral'" dense>
+            mdi-chart-bell-curve
+          </v-icon>
+          <v-icon v-else-if="item.subset_type == 'temporal'" dense>
+            mdi-chart-line
+          </v-icon>
+          {{item.label}}
+        </span>
+      </div>
     </div>
+
   </div>
 </template>
 
 <style scoped>
   .viewer-label {
     display: block;
     float: right;
-    background-color: #c3c3c3c3;
-    width: 24px;
+    background-color: #c3c3c32c;

```

Comment:
Is this change intentional?

Reply:
yes, this adds transparency to the existing gray